backends/graphics/opengl/opengl-sys.h

```diff
@@ -30,6 +30,10 @@
 #include "backends/platform/sdl/sdl-sys.h"
 #endif
 
+#ifdef __MORPHOS__
+#define USE_GLES_MODE 1
+#endif
+
```

Comment:
Wouldn't it be better to add `_opengl_mode=gles` to configure instead?

Reply:
mmm, no sur because MorphOS have only old OpenGL 1.2 (tinygl) so GLES....., need to test unless this

Reply:
I prepare all to test and rebuild it.

Reply:
Ok i could built it (2 hours with my iMac PPC 2.1Ghz....) ! i remove USE_GLES_MODE